shenyu-admin/src/main/resources/mappers/user-role-sqlmap.xml

```diff
@@ -27,46 +27,61 @@
     </resultMap>
 
     <sql id="Base_Column_List">
-        id, date_created, date_updated, user_id, role_id
+        id,
+        date_created,
+        date_updated,
+        user_id,
+        role_id
     </sql>
 
     <select id="selectById" parameterType="java.lang.String" resultMap="BaseResultMap">
-        select
-        <include refid="Base_Column_List"/>
-        from user_role
-        where id = #{id,jdbcType=VARCHAR}
+        SELECT
+               <include refid="Base_Column_List"/>
+          FROM user_role
+        WHERE id = #{id, jdbcType=VARCHAR}
     </select>
 
     <select id="findByUserId" parameterType="java.lang.String" resultMap="BaseResultMap">
-        select
-        <include refid="Base_Column_List"/>
-        from user_role
-        where user_id = #{userId,jdbcType=VARCHAR}
+        SELECT
+               <include refid="Base_Column_List"/>
+          FROM user_role
+        WHERE user_id = #{userId, jdbcType=VARCHAR}
     </select>
 
     <insert id="insert" parameterType="org.apache.shenyu.admin.model.entity.UserRoleDO">
-        insert into user_role (id, date_created, date_updated,
-        user_id, role_id)
-        values (#{id,jdbcType=VARCHAR}, #{dateCreated,jdbcType=TIMESTAMP}, #{dateUpdated,jdbcType=TIMESTAMP},
-        #{userId,jdbcType=VARCHAR}, #{roleId,jdbcType=VARCHAR})
+        INSERT INTO user_role
+                    (id,
+                     date_created,
+                     date_updated,
+                     user_id,
+                     role_id)
+               VALUES
+                    (#{id, jdbcType=VARCHAR},
+                     #{dateCreated, jdbcType=TIMESTAMP},
+                     #{dateUpdated, jdbcType=TIMESTAMP},
+                     #{userId, jdbcType=VARCHAR},
+                     #{roleId, jdbcType=VARCHAR})
```

Comment:
The keywords  are not right-aligned, fileds  are not left-aligned

Reply:
<img width="973" alt="image" src="https://user-images.githubusercontent.com/37730787/159646289-7e7ab4ef-daa5-4907-9ee8-3d4ce984988c.png">
I followed this merged pr to make the changes.